api/src/main/java/org/openmrs/module/commonreports/reports/ChildCareReportManager.java

```diff
@@ -0,0 +1,467 @@
+		_0To60m.setMaxAge(60);
+		_0To60m.setMaxAgeUnit(DurationUnit.MONTHS);
+		
+		// Children seen for the first time
+		SqlCohortDefinition childrenSeenFirstTime = new SqlCohortDefinition();
+		String sql = "SELECT v.patient_id FROM visit v WHERE v.date_started BETWEEN date(:onOrAfter) AND date(:onOrBefore) "
+		        + "AND NOT EXISTS (SELECT 1 FROM visit new_v "
+		        + "WHERE new_v.patient_id = v.patient_id AND new_v.visit_id <> v.visit_id);";
+		childrenSeenFirstTime.setQuery(sql);
+		childrenSeenFirstTime.addParameter(new Parameter("onOrAfter", "On Or After", Date.class));
+		childrenSeenFirstTime.addParameter(new Parameter("onOrBefore", "On Or Before", Date.class));
+		
+		// Children seen for the first time + MUAC measurement
+		Concept muacMeasurementConcept = inizService
+		        .getConceptFromKey("report.childCare.muacMeasurement.numericQuestion.concept");
+		
+		NumericObsCohortDefinition muacMeasured = new NumericObsCohortDefinition();
+		muacMeasured.setQuestion(muacMeasurementConcept);
+		muacMeasured.addParameter(new Parameter("onOrAfter", "On Or After", Date.class));
+		muacMeasured.addParameter(new Parameter("onOrBefore", "On Or Before", Date.class));
+		
+		CompositionCohortDefinition childrenMeasuredForMuac = createCohortComposition(childrenSeenFirstTime, muacMeasured);
+		
+		// Child seen for the first time + weighed / measured
+		NumericObsCohortDefinition weightMeasured = new NumericObsCohortDefinition();
+		weightMeasured
+		        .setQuestion(inizService.getConceptFromKey("report.childCare.weightMeasurement.numericQuestion.concept"));
+		weightMeasured.addParameter(new Parameter("onOrAfter", "On Or After", Date.class));
+		weightMeasured.addParameter(new Parameter("onOrBefore", "On Or Before", Date.class));
+		CompositionCohortDefinition childrenMeasuredWeight = createCohortComposition(childrenSeenFirstTime, weightMeasured);
+		
+		// Children seen for the first time (115 < MUAC <125)
+		NumericObsCohortDefinition muacMeasuredBetween115And125 = new NumericObsCohortDefinition();
+		muacMeasuredBetween115And125.setQuestion(muacMeasurementConcept);
+		muacMeasuredBetween115And125.setValue1(115.0);
+		muacMeasuredBetween115And125.setOperator1(RangeComparator.GREATER_EQUAL);
+		muacMeasuredBetween115And125.setValue2(125.0);
+		muacMeasuredBetween115And125.setOperator2(RangeComparator.LESS_EQUAL);
+		muacMeasuredBetween115And125.addParameter(new Parameter("onOrAfter", "On Or After", Date.class));
+		muacMeasuredBetween115And125.addParameter(new Parameter("onOrBefore", "On Or Before", Date.class));
+		
+		CompositionCohortDefinition childrenMeasuredForMuacBetween115And125 = createCohortComposition(childrenSeenFirstTime,
+		    muacMeasuredBetween115And125);
+		
+		// Children seen for the first time (MUAC <115)
+		NumericObsCohortDefinition muacMeasuredLessThan115 = new NumericObsCohortDefinition();
+		muacMeasuredLessThan115.setQuestion(muacMeasurementConcept);
+		muacMeasuredLessThan115.setValue1(115.0);
```

Comment:
MUAC is recorded in `cm` not `mm`. Should be `15`.
Could you update this in other places too?